.circleci/config.yml

```diff
@@ -110,6 +110,12 @@ jobs:
       - run:
           name: Detekt
           command: ./gradlew detektAll
+      - run:
+          name: Kover HTML
+          command: ./gradlew koverMergedHtmlReport
+      - run:
+          name: Kover XML
+          command: ./gradlew koverMergedXmlReport
```

Comment:
we might want to have this as a separate job, but it will run pretty fast in this case since it runs after tests